controllers/masteruserrecord/sync.go

```diff
@@ -221,7 +221,7 @@ func (s *Synchronizer) alignReadiness() (bool, error) {
 			_, err = notify.NewNotificationBuilder(s.hostClient, s.record.Namespace).
 				WithNotificationType(toolchainv1alpha1.NotificationTypeProvisioned).
 				WithControllerReference(s.record, s.scheme).
-				WithTemplate(notificationtemplates.UserProvisioned.Name).
+				WithTemplate(notificationtemplates.SandboxUserProvisioned.Name).
```

Comment:
Why it's hardcoded to Sandbox instead of relying on the notification env from the config?

Reply:
That is just the name of the template to be stored while creating the Notification. It does actually rely on the notification env when getting the template to fetch - https://github.com/ranakan19/host-operator/blob/emailConfig/controllers/notification/mailgun_notification_delivery_service.go#L77.

Reply:
however the variable `SandboxUserProvisioned` was not needed, so i replaced the variables with constants in [3df1eab](https://github.com/codeready-toolchain/host-operator/pull/754/commits/3df1eab38ec6e59824567e8d6c9d39bd6c43a3ff). This is assuming that the template names remain the same for sandbox and appstudio. If we do however plan on having different templates in future, we can have separate constants for them.

Reply:
The `Sandbox` in the template name confused me. It looks better now.